docker/Dockerfile.festvox

```diff
@@ -0,0 +1,112 @@
+    && unzip -q -d /opt/android/cmdline-tools /tmp/commandlinetools-linux-6200805_latest.zip \
+    && yes Y | /opt/android/cmdline-tools/tools/bin/sdkmanager --install "build-tools;${ANDROID_BUILD_TOOLS_LEVEL}" "platforms;android-${ANDROID_API_LEVEL}" "platform-tools" "ndk;${ANDROID_NDK_VERSION}" \
+    && yes Y | /opt/android/cmdline-tools/tools/bin/sdkmanager --licenses
+
+RUN rm /tmp/*.zip
+
+ENV GRADLE_HOME=/opt/gradle/gradle-$GRADLE_VERSION
+ENV ANDROID_HOME=/opt/android
+ENV ANDROID_NDK_HOME=${ANDROID_HOME}/ndk/${ANDROID_NDK_VERSION}
+ENV ANDROID_NDK=${ANDROID_NDK_HOME}
+ENV ANDROID_NDK_TOOLCHAIN=$ANDROID_NDK/toolchains/llvm/prebuilt/linux-x86_64
+ENV ANDROID_API=$ANDROID_API_LEVEL
+
+# Fetch and prepare Festival & friends
+WORKDIR /usr/local/src
+RUN mkdir -p festival festvox speech_tools
+RUN curl -L https://github.com/grammatek/festival/archive/refs/tags/2.5.2-pre1.tar.gz | \
+    tar xz --no-same-owner --no-same-permissions --strip-components=1 -C festival && \
+    curl -L https://github.com/grammatek/speech_tools/archive/refs/tags/ds-fix-walloc.tar.gz | \
+    tar xz --no-same-owner --no-same-permissions --strip-components=1 -C speech_tools && \
+    curl -L http://festvox.org/packed/festival/2.5/festlex_CMU.tar.gz | \
+    tar xz --no-same-owner --no-same-permissions --strip-components=1 -C festival && \
+    curl -L http://festvox.org/packed/festival/2.5/festlex_OALD.tar.gz | \
+    tar xz --no-same-owner --no-same-permissions --strip-components=1 -C festival && \
+    curl -L http://festvox.org/packed/festival/2.5/festlex_POSLEX.tar.gz | \
+    tar xz --no-same-owner --no-same-permissions --strip-components=1 -C festival && \
+    curl -L http://festvox.org/packed/festival/2.5/voices/festvox_kallpc16k.tar.gz | \
+    tar xz --no-same-owner --no-same-permissions --strip-components=1 -C festival && \
+    curl -L https://github.com/grammatek/festvox/archive/refs/tags/2.8.3-pre1.tar.gz | \
+    tar xz --no-same-owner --no-same-permissions --strip-components=1 -C festvox
+
+ENV ESTDIR /usr/local/src/speech_tools
+ENV FESTVOXDIR /usr/local/src/festvox
+ENV SPTKDIR /usr/local/src/SPTK
+
+# Get SPTK
+RUN cd /usr/local/src && \ 
+    wget http://festvox.org/packed/SPTK-3.6.tar.gz && \
+    tar zxvf SPTK-3.6.tar.gz && \
+    mkdir SPTK && \
+    patch -p0 <festvox/src/clustergen/SPTK-3.6.patch && \
+    cd SPTK-3.6 && \
+    ./configure --prefix=$SPTKDIR && \
+    make && \
+    make install
+
+# Build the Edinburgh Speech Tools
+WORKDIR /usr/local/src/speech_tools
+RUN ./configure && make
+
+# Build Festival
+WORKDIR /usr/local/src/festival
+RUN ./configure && make
+
+# DS: ext/ehmm_patch, for clustergen voices
+RUN git clone https://github.com/lumpidu/unit-selection-festival.git && cp -r unit-selection-festival/ext /usr/local/src/ext
+
+# Build Festvox
+WORKDIR /usr/local/src/festvox
+RUN cp /usr/local/src/ext/ehmm_patch/do_ehmm ./src/ehmm/scripts/
```

Comment:
The ehmm patch is no longer necessary if we use the modified phoneset

Reply:
ok, I'll try to build without the patch